DataFormats/Math/test/CholeskyInvert_t.cu

```diff
@@ -1,5 +1,4 @@
 // nvcc -O3 CholeskyDecomp_t.cu -Icuda-api-wrappers/src/ --expt-relaxed-constexpr -gencode arch=compute_61,code=sm_61 --compiler-options="-Ofast -march=native"
-// add -DDOPROF to run  nvprof --metrics all
```

Comment:
why do you remove this line ?

Reply:
It wasn't my intention, thanks for the fix.